src/main/kotlin/com/github/vol0n/utbotcppclion/services/GeneratorSettings.kt

```diff
@@ -1,30 +1,46 @@
 package com.github.vol0n.utbotcppclion.services
 
-import com.github.vol0n.utbotcppclion.client.GrpcStarter
 import com.intellij.openapi.components.PersistentStateComponent
 import com.intellij.openapi.components.State
-import com.intellij.openapi.components.Storage
+import com.intellij.openapi.components.service
+import com.intellij.openapi.diagnostic.Logger
+import com.intellij.openapi.project.Project
 import com.intellij.util.xmlb.XmlSerializerUtil
-import kotlinx.coroutines.CoroutineExceptionHandler
-import kotlinx.coroutines.CoroutineScope
-import kotlinx.coroutines.Dispatchers
-import kotlinx.coroutines.swing.Swing
 
-@State(name = "Generator settings", storages = [Storage(value = "test_generator_settings.xml")])
+@State(name = "UTBotGeneratorSettings")
 data class GeneratorSettings(
+    @com.intellij.util.xmlb.annotations.Transient
+    val project: Project? = null,
     var generateForStaticFunctions: Boolean = true,
     var useStubs: Boolean = true,
     var useDeterministicSearcher: Boolean = true,
     var verbose: Boolean = false,
-    var timeoutPerFunction: Int = 5,
-    var timeoutPerTest: Int = 20
+    var timeoutPerFunction: Int = 0,
+    var timeoutPerTest: Int = 30
 ) : PersistentStateComponent<GeneratorSettings> {
 
+    @com.intellij.util.xmlb.annotations.Transient
+    val logger = Logger.getInstance(this::class.java)
+
+    init {
+        logger.info("constructor was called: project == $project")
```

Comment:
Is this for debug purpose? If it is, delete it. If not, please specify what constructor was called